precompile/allowlist/test_allowlist.go

```diff
@@ -0,0 +1,218 @@
+// (c) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package allowlist
+
+import (
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ava-labs/subnet-evm/precompile/modules"
+	"github.com/ava-labs/subnet-evm/precompile/testutils"
+	"github.com/ava-labs/subnet-evm/vmerrs"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	AdminAddr   = common.BigToAddress(common.Big1)
+	EnabledAddr = common.BigToAddress(common.Big2)
+	NoRoleAddr  = common.BigToAddress(common.Big3)
```

Comment:
Could we reverse the order of these since they are currently in reverse order of the actual role values they're assigned to?